packages/typespec-test/test/openai_generic/generated/typespec-ts/review/openai-generic.api.md

```diff
@@ -785,6 +785,9 @@ export type Prompt = string | string[] | number[] | number[][];
 // @public
 export type Stop = string | string[];
 
+// @public
+export type Stop_1 = string | string[];
```

Comment:
do we have duplicated stop?

Reply:
tI think this is the same reason of this issue https://github.com/Azure/typespec-azure/issues/1677 
/cc @tadelesh